File: Dev/Code/Community.Entity/Model/Category/CategoryEntity.cs
using System;
using YooPoon.Core.Data;

namespace Community.Entity.Model.Category
{
	public class CategoryEntity : IBaseEntity
	{
		/// <summary>
		/// Id
		/// </summary>
		public virtual int Id { get; set; }
		/// <summary>
		/// 父分类
		/// </summary>
		public virtual CategoryEntity Father { get; set; }
        /// <summary>
        /// 父分类ID 
        /// </summary>
      //  public virtual int FatherId { get; set; }
		/// <summary>
		/// 分类名称
		/// </summary>
		public virtual string Name { get; set; }
		/// <summary>
		/// 分类排序
		/// </summary>
		public virtual int Sort { get; set; }
		/// <summary>
		/// AddUser
		/// </summary>
		public virtual int AddUser { get; set; }
		/// <summary>
		/// AddTime
		/// </summary>
		public virtual DateTime AddTime { get; set; }
		/// <summary>
		/// UpdUser
		/// </summary>
		public virtual int UpdUser { get; set; }
		/// <summary>
		/// UpdTime
		/// </summary>
		public virtual DateTime UpdTime { get; set; }
	}
}

File: Dev/Android/YoopoonMarket/src/com/yoopoon/market/domain/CategoryEntity.java
package com.yoopoon.market.domain;

public class CategoryEntity {

	public int id;
	public String name;
	public int sort;
	public int fatherId;

	public CategoryEntity(int id, String name, int sort, int fatherId) {
		super();
		this.id = id;
		this.name = name;
		this.sort = sort;
		this.fatherId = fatherId;
	}

	@Override
	public String toString() {
		return "CategoryEntity [id=" + id + ", name=" + name + ", sort=" + sort + ", fatherId=" + fatherId + "]";
	}

}
